src/main/java/org/kohsuke/github/GHUser.java

```diff
@@ -36,6 +35,10 @@
  */
 public class GHUser extends GHPerson {
 
+    public List<GHKey> listKeys() throws IOException {
```

Comment:
Do you have a specific reason you used `list` instead of `get`? 

```suggestion
    public List<GHKey> getKeys() throws IOException {
```

Reply:
Ah, I see we use `list` elsewhere.  Got it.